client/quantum_serverless/core/decorators.py

```diff
@@ -317,3 +321,116 @@ def wrapper(*args, **kwargs):
         return wrapper
 
     return decorator
+
+
+ENTRYPOINT_CONTENT = """
+import cloudpickle
+from quantum_serverless import get_arguments, save_result
+
+arguments = get_arguments()
+
+with open("./pickle.pkl", "rb") as file:
+    function = cloudpickle.load(file)
+
+    result = function(**arguments)
+    if result is not None:
+        save_result(result)
+"""
+
+
+def distribute_program(
+    provider: Optional[Any] = None,
+    dependencies: Optional[List[str]] = None,
+    working_dir: Optional[str] = None,
```

Comment:
Multiple distributed programs that are executed at the same time with the same working directory may override the entry point file.

Reply:
Adding random id to the entrypoint may be enough.

Reply:
Fixed in [c27d98f](https://github.com/Qiskit-Extensions/quantum-serverless/pull/908/commits/c27d98f8f7d6706d43f9629638104fd591188502)